### PR TITLE
Add Julia speed section

### PR DESCRIPTION
**Pull request by mfherbst**
Just added what I had on Julia's speed. Happy to drop the vandermonde part if people think it's too much. We can also just have it in the notebook and not talk about it in the presentation. Either way is fine with me.

### CODE REVIEW

File: notebooks/intro.jl [hunk 1]
```diff
@@ -755,9 +1041,9 @@ Unitful = "~1.13.1"
 PLUTO_MANIFEST_TOML_CONTENTS = """
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.8.5"
+julia_version = "1.9.0-rc3"
```
Comment (roflmaostc) [inline notebooks/intro.jl:1047]: should we better use 1.8.5?

Comment (gdalle) [reply notebooks/intro.jl:1047]: I think it's better for us to use 1.9 in order to have fast startup

**Comment by gdalle** [comment]
Currently reviewing the notebook and adding a few tweaks

File: notebooks/intro.jl (hunk 1 above)
Comment (gdalle) [reply notebooks/intro.jl:1047]: Plus 1.9 was literally released this morning

Comment (roflmaostc) [reply notebooks/intro.jl:1047]: haha true!

then we just need to rerun it with 1.9.0 instead of rc3

**Review by gdalle — APPROVED** [review]

**Pull request merged**